.travis.yml

```diff
@@ -0,0 +1,19 @@
+# Documentation: http://docs.travis-ci.com/user/languages/julia
+language: julia
+julia:
+  - 1.0  # Latest LTS release
+  - 1    # Latest stable release
+  - nightly
+os:
+  - linux
+arch:
+  - x64
+  - x86
```

Comment:
32-bit testing is probably unnecessary but I was going to at least start by validating the tests pass